modules/rubiconBidAdapter.js

```diff
@@ -520,6 +521,10 @@ export const spec = {
     if (configUserId) {
       data['ppuid'] = configUserId;
     }
+
+    if (bidderRequest.fledgeEnabled) {
```

Comment:
I think you should look at `ortb2Imp.ext.ae` instead, the pub can enable it only for some slots.